test/streaming/get/test_edith.py

```diff
@@ -41,7 +41,7 @@ async def test_create_synthetic_feeder_does_not_reuse_nmis_by_default(self, feed
         feeder_mrid = "f001"
```

Comment:
I take it back, we probably need at least a couple very simple tests that at least run the mutators - maybe just create a very basic network with one line/transformer and see if it works.